aiohttp_session/__init__.py

```diff
@@ -104,6 +104,12 @@ def __delitem__(self, key):
 SESSION_KEY = 'aiohttp_session'
 STORAGE_KEY = 'aiohttp_session_storage'
 
+async def is_active_session(request):
+    cookie = storage.load_cookie(request)
+    if cookie is not None:
+        return True
+    else:
+        return False
```

Comment:
The idiomatic way of doing this is
```suggestion
    return cookie is not None
```

Reply:
agreed, lol